lockbox-ios/Common/Resources/Strings/es.lproj/Localizable.strings

```diff
@@ -101,7 +101,7 @@
 "recent" = "Reciente";
 
 /* Label for the option sheet action allowing users to sort the logins list by the most recently used logins */
-"recently_used" = "Usado cecientemente";
+"recently_used" = "Usado recientemente";
```

Comment:
Won't this get overwritten during the next import?

Reply:
@joeyg yes, I'll go submit a change to the upstream Pontoon service:
https://pontoon.mozilla.org/es/lockwise-ios/lockwise-ios.xliff/?search=recent&string=199215

Reply:
goood point.